remapping_commands.sh

```diff
@@ -27,22 +27,21 @@ echo "-----------------------1) Flanking sequence generation--------------------
 samtools faidx ../"$oldgenome"
 cut -f1,2 ../"$oldgenome".fai > "$oldgenome".chrom.sizes
 
-# Filter SNPs only
-bcftools filter -i 'TYPE="snp"' ../"$vcffile" -o snps_only.vcf
-
-# Unnecessary if input file is already SNP-filtered and uniq'ed, uncomment if needed: 
-# (also change the snps_only to snps_only.uniq)
-# Get unique variants based on rsIDs (column 3)
-#grep '^#' snps_only.vcf > snps_only.uniq.vcf
-#grep -v '^#' snps_only.vcf | sort -u -k3,3  >> snps_only.uniq.vcf
+#Decompress if required
+if [ "${vcffile: -7}" == ".vcf.gz" ]
+then
+  cp "$vcffile" input.vcf.gz
+  gzip -d -c "$vcffile" > input.vcf
+else
+  cp "$vcffile" input.vcf
+  bgzip -c "$vcffile" > input.vcf.gz
```

Comment:
The previous version was only handling the uncompressed VCF and recompressing it in the next step. Since I start with compressed VCF I thought I would make both files available

Reply:
that's ok for now but I would like to minimise the redundant data. Keep an eye on which is more useful and maybe in the future we can use only one.